frameworks/C/lwan/lwan.dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:19.04
+
+RUN apt update -yqq
+RUN apt install -yqq \
+	git pkg-config build-essential cmake zlib1g-dev \
+	libsqlite3-dev libmariadbclient-dev wget
+
+ADD ./ /lwan
+WORKDIR /lwan
+
+RUN git clone https://github.com/lpereira/lwan --depth 1 && \
```

Comment:
A shallow clone looks quite fragile, and I think that you can achieve the same effect simply by downloading `https://github.com/lpereira/lwan/archive/${commit hash}.zip`.

Reply:
Thanks.  That's probably the reason it's failing in CI.  I've just pushed a new version using your suggestion; seems to be working locally.